Formula/abyss.rb

```diff
@@ -0,0 +1,30 @@
+class Abyss < Formula
+  desc "Genome sequence assembler for short reads"
+  homepage "http://www.bcgsc.ca/platform/bioinfo/software/abyss"
+  url "https://github.com/bcgsc/abyss/releases/download/2.0.2/abyss-2.0.2.tar.gz"
+  sha256 "d87b76edeac3a6fb48f24a1d63f243d8278a324c9a5eb29027b640f7089422df"
+
+  depends_on "boost" => :build
+  depends_on "google-sparsehash" => :build
+  depends_on :mpi => :cc
+
+  resource("testdata") do
+    url "http://www.bcgsc.ca/platform/bioinfo/software/abyss/releases/1.3.4/test-data.tar.gz"
+    sha256 "28f8592203daf2d7c3b90887f9344ea54fda39451464a306ef0226224e5f4f0e"
+  end
+
+  def install
+    system "./configure", "--enable-maxk=128", "--prefix=#{prefix}",
+           "--disable-dependency-tracking", "--disable-silent-rules",
+           "--disable-openmp"
```

Comment:
Please add `needs :openmp` and `--enable-openmp`. The majority of the ABySS pipeline is parallelized using OpenMP. Only the very first stage is parallelized using MPI.

Reply:
```
  option "with-openmp", "Enable OpenMP multithreading"
```
was used by no one in the last 90 days so why would we make it the default?

Reply:
Because otherwise the tool would not be parallelized, which would make assembly of large genomes terribly slow. Users use the default because it's the default, and they do not know better.

Reply:
And you're OK with no opt-out?

Reply:
I’ll defer to Shaun on this once since I’ve only used abyss once or twice. I’m ok with adding `needs :openmp` as a default.

Reply:
:+1:

Reply:
Yes, `needs :openmp` as the default with no opt-out is fine.